bin/plugin/commands/release.js

```diff
@@ -71,34 +71,64 @@ async function runReleaseBranchCreationStep(
 ) {
 	let version, releaseBranch, versionLabel;
 	await runStep( 'Creating the release branch', abortMessage, async () => {
+		let parsedMajorVersion;
+
+		const getComponentsOfRC = ( ver ) => ver.match( /(.+-rc\.)([0-9]+)/ );
 		const packageJsonPath = gitWorkingDirectoryPath + '/package.json';
 		const packageJson = readJSONFile( packageJsonPath );
-		const nextMajor = getNextMajorVersion( packageJson.version );
-		const parsedMajorVersion = semver.parse( nextMajor );
+		const rcComponents = getComponentsOfRC( packageJson.version );
+		const isRC = !! rcComponents;
```

Comment:
While `isRC` is accurate naming, it's prone to confusion with the (user-supplied) `isRC` bool flag in `releasePlugin()`. Let's maybe rename this one. `packageVersionIsRC` maybe?

Reply:
or `isPackageVersionRC`? That's more predicate-style.

Reply:
Ohh, I missed that bool flag. I think `isPackageVersionRC` is a good name 👍🏻